ruby/private/providers.bzl

```diff
@@ -45,3 +45,23 @@ def get_transitive_deps(deps):
         deps,
         transitive = [dep[RubyFilesInfo].transitive_deps for dep in deps],
     )
+
+def get_bundle_envs(envs, deps):
```

Comment:
Do we want to iterate over transitive dependencies?

Reply:
good catch!